synapse/lib/snap.py

```diff
@@ -123,13 +151,23 @@ async def iterStormPodes(self, text, opts=None, user=None):
 
         self.core._logStormQuery(text, user)
 
+        scrubber = None
+        # NOTE: This option is still experimental and subject to change.
+        if opts.get('scrub') is not None:
```

Comment:
is this intended mainly for the export use case; or for any remote storm call?

Reply:
Intended for potential use in any storm call.